docs/sphinx/source/whatsnew/v0.11.2.rst

```diff
@@ -20,7 +24,10 @@ Bug fixes
 Documentation
 ~~~~~~~~~~~~~
 * Edited docstrings for :py:func:`~pvlib.pvsystem.dc_ohms_from_percent` and
-  :py:func:`~pvlib.pvsystem.dc_ohmic_losses` for clarity. (:issue:`1601`, :pull:`2229`)
+  :py:func:`~pvlib.pvsystem.dc_ohmic_loss` for clarity. (:issue:`1601`, :pull:`2229`)
```

Comment:
This should be `losses`, not `loss`.  Somehow this change keeps sneaking into PRs, I am not sure why.